src/main/scala/com/databricks/spark/sql/perf/results.scala

```diff
@@ -106,37 +106,37 @@ case class Failure(className: String, message: String)
 
 // KEEP ARGUMENTS SORTED BY NAME.
 // It simplifies lookup when checking if a parameter is here already.
-case class MLParams(
+class MLParams(
```

Comment:
Discussed offline with Sid: He will send a PR to stop using MLParams and start using BenchmarkResult.parameters instead.